docs/pages/includes/insecure-certificate.mdx

```diff
@@ -3,6 +3,7 @@
 
   - In Safari's "This Connection Is Not Private" page, click "Show Details," then click "visit this website."
   - In Firefox, click "Advanced" from the warning page, then click "Accept the Risk and Continue."
-  - In Chrome's warning page, type `thisisunsafe` to ignore certificate validation for the Teleport Web UI.
+  - In Chrome's warning page, click "Advanced" then "Proceed to `URL`"
+    - If a server previously acessible from this URL set an HSTS header, the "Proceed to" link will not display. Type `thisisunsafe` to ignore the HSTS header and proceed.
```

Comment:
```suggestion
    - If a server previously accessible from this URL set an HTTP Strict Transport Security (HSTS) header, the "Proceed to" link will not display. Type `thisisunsafe` to ignore the HSTS header and proceed.
```
Spelling, also expanding the abbreviation in case the reader is unfamiliar

Reply:
The shame of this spelling error will haunt me until I install a spellchecker in vscode.